portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,25 +68,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     @Override
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        // Get the input from the user.
-        String comment = request.getParameter(INPUT_NAME);        
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {   
+        // Get the comment input from the user.
+        String comment = request.getParameter(COMMENT_INPUT);
         
         Entity taskEntity = new Entity(DATASTORE_LABEL);
-        taskEntity.setProperty(INPUT_NAME, comment);
+        taskEntity.setProperty(COMMENT_INPUT, comment);
 
         // Store the user comment in datastore.
         datastore.put(taskEntity);
 
-        response.sendRedirect(HOME_URL);
+        response.sendRedirect(COMMENTS_URL);
     }
 
     /**
-     * Converts a Java ArrayList<String> into a JSON string using Gson.  
+     * Converts a Java List<String> into a JSON string using Gson.  
```

Comment:
nit: maybe update this comment to say a List of comments (each represented as a Map<String, String>